src/graph/TruncateSpaceExecutor.cpp

```diff
@@ -0,0 +1,199 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+#include "graph/TruncateSpaceExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+TruncateSpaceExecutor::TruncateSpaceExecutor(Sentence *sentence,
+                                         ExecutionContext *ectx)
+        : Executor(ectx) {
+    sentence_ = static_cast<TruncateSpaceSentence*>(sentence);
+}
+
+Status TruncateSpaceExecutor::prepare() {
+    spaceName_ = sentence_->spaceName();
```

Comment:
why here no need check spaceName_ is nullptr?

Reply:
The parser has guaranteed.